torch/autograd/profiler.py

```diff
@@ -552,9 +554,9 @@ def createFunctionEventForMemoryEvents(evt):
 
 
 class record_function(_ContextDecorator):
-    """Context manager/function decorator that adds a label to a block of
-    Python code (or function) when running autograd profiler. It is
-    useful when tracing the code profile.
+    """Context manager that adds a label to a code block/function when running autograd profiler.
```

Comment:
why did we remove function decorator here?

Reply:
Same reason  above